product_barcode_generator/data/barcode_rule.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <data noupdate="1">
-        <record model="barcode.rule" id="beesdoo_product_barcode_rule">
-            <field name="name">Beescoop Product Barcodes</field>
+        <record model="barcode.rule" id="product_barcode_rule">
+            <field name="name">product_barcode_generator</field>
```

Comment:
I don't know what this name is. Should it be tidier/more informative? Do users see this string?